cms/appresolver.py

```diff
@@ -209,6 +211,8 @@ def get_app_patterns():
         current_site = Site.objects.get_current()
     except Site.DoesNotExist:
         current_site = None
+    except OperationalError:
+        return []
```

Comment:
It's the only error we get if the database is not ready (e.g.: with no tables in it)